xml/src/test/resources/configs/docs/getting-started.xml

```diff
@@ -22,6 +22,7 @@
   <!-- tag::gettingStarted[] -->
   <cache alias="foo"> <!--1-->
     <key-type>java.lang.String</key-type> <!--2-->
+    <value-type>java.lang.String</value-type> <!--2-->
```

Comment:
Same callout <2> for both lines. Not sure if the added line is required. But I see that it was added to master some time back. So I guess we need to keep it for consistency.